frontend/layouts/user.vue

```diff
@@ -63,4 +66,17 @@ footer{
   border-top: 2px solid #42A5F5;
 }
 
+.auth-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 25vw;
```

Comment:
Trochę risky bo jak ktoś ma inny ekran to się może brzydko skalować, lepiej ustawić jakąś sztywną wartość